src/script/calling/CallingRepository.ts

```diff
@@ -1208,13 +1208,13 @@ export class CallingRepository {
       // When a second call arrives in the same conversation, we need to clean that call first
       this.removeCall(storedCall);
     }
-    const canRing = !conversationEntity.showNotificationsNothing() && shouldRing && this.isReady;
+    const canRing = !conversation.showNotificationsNothing() && shouldRing && this.isReady;
     const selfParticipant = new Participant(this.selfUser, this.selfClientId);
     const isVideoCall = hasVideo ? CALL_TYPE.VIDEO : CALL_TYPE.NORMAL;
     const isMuted = Config.getConfig().FEATURE.CONFERENCE_AUTO_MUTE && conversationType === CONV_TYPE.CONFERENCE;
     const call = new Call(
       this.parseQualifiedId(userId),
-      conversationId,
+      conversation.qualifiedId,
```

Comment:
all conversations will now have a qualifiedId right? (just checking)

Reply:
All calls will have a qualified conversation id attached to it